Added matheuspoleza/cookiecutter-angular2

matheuspoleza opened the pull request.

luzfcb commented on README.rst:
```diff
@@ -331,6 +332,7 @@ HTML
 .. _`funkload-friendly`: https://github.com/tokibito/funkload-friendly
 .. _`cookiecutter-python-app`: https://github.com/mdklatt/cookiecutter-python-app
 .. _`cookiecutter-reveal.js`: https://github.com/keimlink/cookiecutter-reveal.js
+.. _`cookiecutter-angular2`: https://github.com/matheuspoleza/cookiecutter-angular2.js
```
your repository name is `cookiecutter-angular2` and not `cookiecutter-angular2.js`

https://github.com/matheuspoleza/cookiecutter-angular2

matheuspoleza replied:
:disappointed: my mistake!

hackebrot commented:
Hi @matheuspoleza, thanks for submitting a PR 🙇 

Your template's license file is empty, although you are stating that you are using the MIT license for the project on the readme. Also the travis badge links to something completely different.

matheuspoleza commented:
Hey, @hackebrot. Thanks for replied. 

I made the changes:
- [x] Added LICENSE.md 
- [x] Added CONTRIBUTING.md 
- [x] Fixed travis build status

Can u check this out?

hackebrot commented:
https://github.com/matheuspoleza/cookiecutter-angular2/blob/master/CONTRIBUTING.md#tests-arent-optional 😉

matheuspoleza commented:
@hackebrot didn't get it. Its good?

hackebrot commented:
Well, you clearly copy-pasted an existing contributing guide from a Python project. The guide talks about versioning (your template doesn't have one), it states that documentation and tests are mandatory (don't have neither on your template) and it talks about Python coding conventions, but your template is not using any Python at all.

That being said, having a CONTRIBUTING guide that is misleading is way worse than having none, IMHO.

matheuspoleza commented:
@hackebrot sorry, I made things too fast. Now its ok: https://github.com/matheuspoleza/cookiecutter-angular2/blob/master/CONTRIBUTING.md
